sequencer/api/merklized_state.toml

```diff
@@ -0,0 +1,4 @@
+[route.getfeebalance]
+PATH = ["/fee_balance/:address"]
```

Comment:
As a convention we usually use hyphens in URL segments eg `fee-balance`

Reply:
done in d19a139fb2599f6147bd6d5413adc5f0891338e3